_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -339,28 +301,17 @@ This [HTML document][] has a [keyboard event][] [dispatched][] to an [event targ
 
 #### Inapplicable Example 1
 
-This document is not an [HTML document][].
-
-```html
-<svg height="100" width="100">
-  <circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red" />
-  Sorry, your browser does not support inline SVG.  
-</svg>
-```
-
-#### Inapplicable Example 2
-
-This [HTML document][] has a [keyboard event][] [dispatched][] to an [event target][] with the attribute `key` being a [printable character][] and the method `getModifierState` returning `true`.
+This [HTML document][] has a [keyboard event][] [dispatched][] to an [event target][] but it only causes [changes in content][] if the event's attribute `key` is a [non-printable character][non-printable characters].
```

Comment:
I'm actually not sure this example is working in its current state.
I don't think that pressing Ctrl alone triggers a keyboard event. And given that the default key is `''`, pressing Ctrl+anything will fail the test on https://github.com/act-rules/act-rules.github.io/pull/873/files/5ba7404896460d5f279b3f5d8a7496f61dbb4061..225df1c2a17404420a433f500901097f20401d02#diff-b6da7667926ff757a971a635c511cccbR18

We should replace this with an example with Escape or similar key.

Reply:
Made the suggested change.